src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -188,12 +194,37 @@ class RobotContainer {
     m_autoCommandChooser.addOption("Backup 2s and Look Cool Autonomous", spinAuto)
     m_autoCommandChooser.addOption("No auto (DON'T PICK)", noAuto)
 
-    SmartDashboard.putData("Auto mode", m_autoCommandChooser)
 
+    m_shooterCommandChooser.setDefaultOption("Fixed Speed", FixedShooterSpeed(shooter,{ 100.0 }))
+    m_shooterCommandChooser.addOption("PIDShooter",ShooterPID(shooter,{ 100.0 }))
+    SmartDashboard.putData("Auto mode", m_autoCommandChooser)
+    SmartDashboard.putData("Shooter Mode", m_shooterCommandChooser)
   }
 
   fun getAutonomousCommand(): Command {
     // Return the selected command
     return m_autoCommandChooser.getSelected()
+    /*
+    var autoVoltageConstraint = DifferentialDriveVoltageConstraint(
+            SimpleMotorFeedforward(Constants.ksVolts),
+            Constants.kvVoltSecondsPerMeter,
+            Constants.kaVoltSecondsSquaredPerMeter
+        )
+    //        Constants.kDriveKinematics,
+	//10)
+
+	val config: TrajectoryConfig = TrajectoryConfig(
+		Constants.kMaxSpeedMetersPerSecond,
+		Constants.kMaxAccelerationMetersPerSecondSquared
+	)
+	.setKinematics(Constants.kDriveKinematics)
+	.addConstraint(autoVoltageConstraint)
+	
+	val trajectoryJSON: String = "PathWeaver/Paths/Unnamed.path"
+	val trajectoryPath: Path = Filesystem.getDeployDirectory().toPath().resolve(trajectoryJSON)
+	val trajectory: Trajectory = TrajectoryUtil.fromPathweaverJson(trajectoryPath)
+     */
```

Comment:
Do we need this code?

Reply:
Yes, it's a start on the PathWeaver code.